src/components/ProjectCard/ProjectCard.module.css

```diff
@@ -0,0 +1,109 @@
+body {
+  background-color: black;
+}
+
+h5 {
+  color: rgb(51, 255, 0);
+}
+
+.workcard {
+  width: 19rem;
+  height: 370px;
```

Comment:
Please use max or min width or use vw, vh, or pass width and height in percentage, never pass static width or height in percentage

Reply:
done